usr/bin/steamos-session

```diff
@@ -6,7 +6,10 @@ export HOMETEST_USER=steam
 export HOMETEST_DESKTOP_USER=desktop
 export HOMETEST_DESKTOP_SESSION=gnome
 
-/usr/bin/steamos/set_hd_mode.sh >> $HOME/set_hd_mode.log
+# Add our bin directory with the set_hd_mode and dpkg-query replacement scripts
+export PATH=/usr/share/steamos-compositor-plus/bin:${PATH}
+
+set_hd_mode.sh >> $HOME/set_hd_mode.log
```

Comment:
does moving `set_hd_mode.sh` break the debian packaging?

Reply:
The Debian packaging is currently broken, but it doesn't have issues with the scripts. It currently can't put the binaries it builds in the correct place, but that was already the case before these changes. I'll look into that some other time

Reply:
ok, I thought you had said you had managed to build it, but if it is already broken, no matter. Maybe we should just remove the debian directory?

Reply:
I think we should keep it for now. It isn't that hard to fix. The rules file will need to be changed to put the files in the correct directory after compiling.

Reply:
I've updated the debian package in pull request #11